orchestra/communication/slack.py

```diff
@@ -69,17 +69,14 @@ def add_worker_to_project_team(worker, project):
     slack = OrchestraSlackService()
     try:
         slack_user_id = worker.slack_user_id
-        response = slack.groups.invite(
+        slack.conversations.invite(
             project.slack_group_id, slack_user_id)
-        if not response.body.get('already_in_group'):
-            welcome_message = (
-                '<@{}> has been added to the team. '
-                'Welcome aboard!').format(slack_user_id)
-            slack.chat.post_message(project.slack_group_id, welcome_message)
+        welcome_message = (
```

Comment:
if you care I think it's `if response.body.get('error') != 'already_in_channel'`

Reply:
Thanks!